packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -3485,7 +3487,7 @@ export default abstract class SqlIntegration
     return filterValues;
   }
   // Get a Fact Table CTE for multiple fact metrics that all share the same fact table
-  private getFactTableCTE({
+  private getFactMetricCTE({
```

Comment:
Renamed this, so it would be more specific to building the fact table CTE for metrics.